### PR TITLE
Add in stderr to logs

### PR DESCRIPTION
**Pull request by mueschm**
## Overview
Make sure we pipe stderr to stderr

## Changes
Make sure we pipe stderr to stderr

## Tests
Showing errors it did not show before
image

### CODE REVIEW

File: src/commands/dev/index.ts [hunk 1]
```diff
@@ -175,30 +175,34 @@ export class UpProcessManager {
     }
 
     const service_colors = new Map<string, chalk.Chalk>();
-    this.compose_process.stdout?.on('data', (data) => {
-      if (this.is_exiting) {
-        return;
-      }
+    const createHandleStream = (output_func: (message: string) => void) => {
```
Comment (tjhiggins) [inline src/commands/dev/index.ts:178]: I also noticed that we don't pipe stderr here:
```ts
const compose_process = DockerComposeUtils.dockerCompose(compose_args,
      { stdout: 'pipe', stdin: 'ignore', detached: !this.is_windows });
```

Comment (tjhiggins) [reply src/commands/dev/index.ts:178]: Oh I guess the default is pipe so maybe not a big deal, but confusing to read

File: src/commands/dev/index.ts [hunk 2]
```diff
@@ -175,30 +175,34 @@ export class UpProcessManager {
...
-            service_colors.set(service, chalk.rgb(rand(), rand(), rand()));
-          }
+            if (!service_colors.get(service)) {
+              service_colors.set(service, chalk.rgb(rand(), rand(), rand()));
+            }
 
-          const color = service_colors.get(service) as chalk.Chalk;
-          console.log(color(service + '| ') + newLine);
+            const color = service_colors.get(service) as chalk.Chalk;
+            output_func(color(service + '| ') + newLine);
```
[the remark this answers is not in the document]
Comment (mueschm) [reply src/commands/dev/index.ts:199]: So docker logs does not do this, so just tried to copy what they did.

File: src/commands/dev/index.ts (hunk 1 above)
Comment (mueschm) [reply src/commands/dev/index.ts:178]: Fair I can make that more obvious so no one spends time going down some kind of wrong rabbit hole.

File: src/commands/dev/index.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (TylerAldrich) [reply src/commands/dev/index.ts:199]: Oh interesting, I thought they did but I think what I'm actually thinking of is docker build errors

**Pull request merged**